services/mapper.go

```diff
@@ -271,13 +334,8 @@ func TraceOps(
 	return ops
 }
 
-// Erc20Ops returns a list of erc20 operations parsed from the log from a transaction receipt
-func Erc20Ops(
```

Comment:
I think we need to leave this function in this format. please revert the syntax.